devfile.yaml

```diff
@@ -1,26 +1,44 @@
 specVersion: 0.0.1
 name: chectl
 projects:
-  - name: chectl
+  - 
+    name: chectl
     source:
       type: git
       location: 'https://github.com/che-incubator/chectl.git'
 components:
-  - alias: theia-ide
+  - 
+    id: eclipse/che-theia/next
     type: cheEditor
-    id: eclipse/che-theia/1.0.0
-  - type: chePlugin
+  - 
     id: eclipse/che-machine-exec-plugin/0.0.1
+    type: chePlugin
+  - 
+    alias: git-devtools
```

Comment:
can we avoid the `alias` here since it's not used in the commands

Reply:
not used by commands but it is display nicely with this name on the `my workspace` view

Reply:
Ok makes sense